src/user/selectors/index.js

```diff
@@ -0,0 +1,3 @@
+import { createSelector } from 'reselect';
+
+export const user = createSelector(state => state.userReducer, userReducer => userReducer.user);
```

Comment:
I think that we should change how user state is named in rootReducer. It should be just `user` - without `Reducer` part.

Reply:
yup, agreed